pingdom-checks/community-care.ping

```diff
@@ -60,3 +60,54 @@ pingdom save-check \
   -a group="community-care" \
   -a userids_csv="$NO_USERS" \
   -a integrationids_csv="$COMMUNITY_CARE_BACKEND_SLACK_ID"
+
+  pingdom save-check \
+    --template https-public-200 \
+    -a name=production-community-care-eligibility-backend-health \
+    -a host=api.va.gov \
+    -a url="/services/community-care/dependency/health" \
+    -a port="443" \
+    -a responsetime_threshold="30000" \
+    -a resolution="1" \
+    -a group="community-care" \
+    -a userids_csv="$NO_USERS" \
+    -a integrationids_csv="$COMMUNITY_CARE_BACKEND_SLACK_ID"
```

Comment:
We want to update the old /v0/health to /dependency/health. To ensure there is no discontinuity between CCE and Pingdom, we decided to add endpoint /dependency/health and leave /v0/health for now and remove the outdated endpoint /v0/health later